configureWorkspace/configure.ts

```diff
@@ -379,7 +379,6 @@ services:
 }
 
 function genDockerIgnoreFile(service: string, platformType: string, os: string, port: string) {
-    // TODO: Add support for other platform types
```

Comment:
Is this 'TODO' already done or tracked elsewhere?

Reply:
Doesn't seem to apply anymore.